lib/git-fastclone/version.rb

```diff
@@ -2,5 +2,5 @@
 
 # Version string for git-fastclone
 module GitFastCloneVersion
-  VERSION = '1.2.7'.freeze
+  VERSION = '1.2.8'.freeze
```

Comment:
Probably 1.3.0 for net new functionality (rather than just being a bug fix).

Reply:
Good point!

```suggestion
  VERSION = '1.3.0'.freeze
```